recipes/rb-minima/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "minima" %}
+{% set version = "2.5.1" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: 520e52bc631fb16cbb8100660f6caa44f97859e2fa7e397d508deb18739567be
+
+build:
+  noarch: generic
+  number: 0
+  script:
+    - gem install --norc -l -V --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+  skip: {{ win }}
+
+requirements:
+  host:
+    - ruby
+  run:
+    - ruby
+    - rb-jekyll >=3.5,<5.0
+    - rb-jekyll-feed >=0.9,<1
+    - rb-jekyll-seo-tag >=2.1,<3
+
+test:
+  commands:
+    - gem list | grep -s {{ name }}
```

Comment:
Minima is only a theme, so there was nothing obvious to test other than seeing if the gem was installed correctly.